.eslintrc

```diff
@@ -14,20 +14,342 @@
+		"guard-for-in": "off",
+		"handle-callback-err": "error",
+		"id-blacklist": "error",
+		"id-length": "off",
+		"id-match": "error",
+		"implicit-arrow-linebreak": [
+			"error",
+			"beside"
+		],
+		"indent": [
+			"error",
+			"tab"
+		],
+		"init-declarations": "off",
+		"jsx-quotes": "error",
+		"key-spacing": "warn",
+		"keyword-spacing": "warn",
+		"line-comment-position": "off",
+		"linebreak-style": [
+			"error",
+			"unix"
+		],
+		"lines-around-comment": "error",
+		"lines-around-directive": "error",
+		"lines-between-class-members": "error",
+		"max-depth": "off",
+		"max-len": "off",
+		"max-lines": "off",
+		"max-nested-callbacks": "error",
+		"max-params": "off",
+		"max-statements": "off",
+		"max-statements-per-line": "off",
+		"multiline-comment-style": "off",
+		"multiline-ternary": [
+			"error",
+			"always-multiline"
+		],
+		"new-parens": "off",
+		"newline-per-chained-call": "error",
+		"no-alert": "off",
+		"no-array-constructor": "error",
+		"no-await-in-loop": "warn",
+		"no-bitwise": "off",
+		"no-buffer-constructor": "error",
+		"no-caller": "error",
+		"no-confusing-arrow": "error",
+		"no-constant-condition": [
+			"error",
+			{
+			"checkLoops": false
+			}
+		],
+		"no-control-regex": "off",
+		"no-continue": "off",
+		"no-div-regex": "off",
+		"no-duplicate-imports": "error",
+		"no-else-return": "off",
+		"no-empty": "off",
+		"no-empty-function": "off",
+		"no-eq-null": "error",
+		"no-eval": "error",
+		"no-extend-native": "error",
+		"no-extra-bind": "error",
+		"no-extra-label": "error",
+		"no-extra-parens": "off",
+		"no-floating-decimal": "error",
+		"no-implicit-globals": "error",
+		"no-implied-eval": "error",
+		"no-inline-comments": "off",
+		"no-invalid-this": "off",
+		"no-iterator": "error",
+		"no-label-var": "error",
+		"no-labels": "off",
+		"no-lone-blocks": "error",
+		"no-lonely-if": "error",
+		"no-loop-func": "error",
+		"no-magic-numbers": "off",
+		"no-mixed-operators": "off",
+		"no-mixed-requires": "error",
+		"no-multi-assign": "off",
+		"no-multi-spaces": "error",
+		"no-multi-str": "error",
+		"no-multiple-empty-lines": "warn",
+		"no-native-reassign": "error",
+		"no-negated-condition": "off",
+		"no-negated-in-lhs": "error",
+		"no-nested-ternary": "off",
+		"no-new": "error",
+		"no-new-func": "error",
+		"no-new-object": "error",
+		"no-new-require": "error",
+		"no-new-wrappers": "error",
+		"no-octal-escape": "error",
+		"no-param-reassign": "off",
+		"no-path-concat": "error",
+		"no-plusplus": "off",
+		"no-process-env": "error",
+		"no-process-exit": "error",
+		"no-proto": "error",
+		"no-prototype-builtins": "off",
+		"no-restricted-globals": "error",
+		"no-restricted-imports": "error",
+		"no-restricted-modules": "error",
+		"no-restricted-properties": "error",
+		"no-restricted-syntax": "error",
+		"no-return-assign": "off",
+		"no-return-await": "error",
+		"no-script-url": "error",
+		"no-self-compare": "error",
+		"no-sequences": "error",
 		"no-shadow": "off",
```

Comment:
I can't assess how impactful/useful this would be for Zotero, but this one has saved me on occasion from some confusion.